src/mem/phymem.c

```diff
@@ -51,3 +51,7 @@ void phymem_free(void *page, size_t page_number) {
 	munmap(page, PAGE_SIZE() * page_number);
 	page_free(__phymem_allocator, page, page_number);
 }
+
+void phymem_only_free(void *page, size_t page_number) {
```

Comment:
Seems like you encounter phymem_free behavior that calles munmap for freeing memory. That is not correct, so was fixed recently https://github.com/embox/embox/pull/1084/commits/d066c061a36a9d2f012d4cbed4d50346163607f8

Could you rebase this branch on top of master or merge master? You'll get fix in phymem_free and this function could be removed